### ISSUE TITLE
Is this a scam?

### ISSUE THREAD

**Issue by danielkrajnik** [mastodon/mastodon#27706]
### Pitch

I'm getting started with self hosting mastodon. Even though the README mentions docker compose documentation doesn't include any page on actually using it. There is a page for [installing from source](https://docs.joinmastodon.org/admin/install/) and referrals to various cloud providers for their one-click solutions...

The docs propose overly complicated steps (e.g. manipulating iptables directly instead of using ufw) seemingly to discourage the users.

I've observed a similar pattern trying to [set up lemmy yesterday](https://github.com/LemmyNet/lemmy/issues/4132).

So, my question is - is this a scam? Do you want everyone to use mastodon.social? Are you just another twitter? Are you going to add opt-out telemetry, take up VC funding [continues to escalate aimlessly...]

### Motivation

a

**Comment by Znuff** [comment]
You're clearly too much of a moron to run such a project.

Go back to Twitter and Elon Musk.

**Comment by Daniel15** [comment]
The whole project is open source and there's thousands of servers currently deployed, most of which would have no direct connection to the Mastodon developers, so I'm not sure how it could possibly be considered a "scam".

**Comment by danielkrajnik** [comment]
> most of which would have no direct connection to the Mastodon developers

@Daniel15 Exactly, so that's my main concern right now. How sure are you about that?

**Comment by Znuff** [comment]
Please take your meds, my dude.

**Comment by trwnh** [comment]
https://github.com/mastodon/documentation

**Comment by Sieboldianus** [comment]
Just for anyone looking, here is the missing documentation:
https://du.nkel.dev/blog/2023-12-12_mastodon-docker-rootless/
